LoopKitUI/LoopUIPlugin.swift

```diff
@@ -6,9 +6,19 @@
 //  Copyright © 2019 LoopKit Authors. All rights reserved.
 //
 
-import Foundation
-
-public protocol LoopUIPlugin {
+public protocol PumpManagerUIPlugin {
```

Comment:
What are we gaining by splitting the LoopUIPlugin protocol into different protocols? Plugins are just packages; we could have plugins vend many different things, like multiple services.  I was thinking the Tidepool plugin might vend logging, data storage, and support functionality, for example.

Reply:
If it is one protocol then all plugins need to implement all of the methods. It doesn't really make sense for the Dexcom plugin to have to implement:
```
var serviceType: ServiceUI.Type? { get }
```
This way a plugin can implement only what it supports and nothing more.

Reply:
Is this good?

Reply:
Yes, I think this works; as long as a plugin can still vend multiple types; and I think it can by conforming to multiple protocols here.